src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -897,6 +904,12 @@ public sealed partial class DeploymentTargetAnnotation : IResourceAnnotation
     {
         public DeploymentTargetAnnotation(IResource target) { }
 
+        [System.Diagnostics.CodeAnalysis.Experimental("ASPIRECOMPUTE001")]
+        public IComputeEnvironmentResource? ComputeEnvironment { get { throw null; } set { } }
+
+        [System.Diagnostics.CodeAnalysis.Experimental("ASPIRECOMPUTE001")]
+        public IContainerRegistry? ContainerRegistryInfo { get { throw null; } set { } }
```

Comment:
Why is this ContainerRegistryInfo and not ContainerRegistry @captainsafia ?

cc @eerhardt